README.md

```diff
@@ -1278,131 +1268,33 @@ terragrunt = {
-    interpolation functions:
-
-    ```hcl
-    # terraform.tfvars
-    terragrunt = {
-      remote_state {
-        backend = "s3"
-        config {
-          bucket  = "my-terraform-state"
-          key     = "${path_relative_to_include()}/terraform.tfstate"
-          region  = "us-east-1"
-          encrypt = true
-        }
-      }
-    }
-    ```
-
-1. Remove any `lock { ... }` blocks from your Terragrunt configurations, as these are no longer supported.
-
-    If you were storing remote state in S3 and relying on DynamoDB as a locking mechanism, Terraform now supports that
-    natively. To enable it, simply add the `lock_table` parameter to your S3 backend configuration. If you configure
-    your S3 backend using Terragrunt, then Terragrunt will automatically create the `lock_table` for you if that table
-    doesn't already exist:
-
-    ```hcl
-    # terraform.tfvars
-    terragrunt = {
-      remote_state {
-        backend = "s3"
-        config {
-          bucket  = "my-terraform-state"
-          key     = "${path_relative_to_include()}/terraform.tfstate"
-          region  = "us-east-1"
-          encrypt = true
-
-          # Tell Terraform to do locking using DynamoDB. Terragrunt will automatically create this table for you if
-          # it doesn't already exist.
-          lock_table = "my-lock-table"
-        }
-      }
-    }
-    ```
-
-    **NOTE**: We recommend using a completely new lock table name and NOT reusing the lock table from older versions of
-    Terragrunt, as that older table had a different structure than what Terraform expects, and Terragrunt will not
-    automatically recreate it.
-
-    If you would like Terraform to automatically retry locks like Terragrunt did (this is particularly useful when
-    running Terraform as part of an automated script, such as a CI build), you use an `extra_arguments` block:
-
-    ```hcl
-    # terraform.tfvars
-    terragrunt = {
-      remote_state {
-        backend = "s3"
-        config {
-          bucket  = "my-terraform-state"
-          key     = "${path_relative_to_include()}/terraform.tfstate"
-          region  = "us-east-1"
-          encrypt = true
-
-          # Tell Terraform to do locking using DynamoDB. Terragrunt will automatically create this table for you if
-          # it doesn't already exist.
-          lock_table = "my-lock-table"
-        }
-      }
-
-      terraform {
-        # Force Terraform to keep trying to acquire a lock for up to 20 minutes if someone else already has the lock
-        extra_arguments "retry_lock" {
-          commands = [
-            "init",
-            "apply",
-            "refresh",
-            "import",
-            "plan",
-            "taint",
-            "untaint"
-          ]
-
-          arguments = [
-            "-lock-timeout=20m"
-          ]
-        }
-      }
-    }
-    ```
+##### Migrating from Previous Versions of Terragrunt
```

Comment:
We have several links (including on the releases page) pointing to the old heading, `Migrating from Terragrunt v0.11.x and Terraform 0.8.x and older`. I think we should keep that link as-is, both to make it easier to find with `CTRL+F` and to keep those links backwards compatible.